tests/components/ffmpeg/test_init.py

```diff
@@ -104,25 +108,37 @@ async def test_setup_component_test_register(hass: HomeAssistant) -> None:
     with assert_setup_component(1):
         await async_setup_component(hass, ffmpeg.DOMAIN, {ffmpeg.DOMAIN: {}})
 
-    hass.bus.async_listen_once = MagicMock()
     ffmpeg_dev = MockFFmpegDev(hass)
+    ffmpeg_dev._async_stop_ffmpeg = AsyncMock()
+    ffmpeg_dev._async_start_ffmpeg = AsyncMock()
     await ffmpeg_dev.async_added_to_hass()
 
-    assert hass.bus.async_listen_once.called
-    assert hass.bus.async_listen_once.call_count == 2
+    hass.bus.async_fire(EVENT_HOMEASSISTANT_START)
+    await hass.async_block_till_done()
+    assert len(ffmpeg_dev._async_start_ffmpeg.mock_calls) == 2
+
+    hass.bus.async_fire(EVENT_HOMEASSISTANT_STOP)
+    await hass.async_block_till_done()
+    assert len(ffmpeg_dev._async_stop_ffmpeg.mock_calls) == 2
```

Comment:
This should be refactored a bit more in the future, but thats out of scope for this PR.